cmd/catalyst-node/catalyst-node_test.go

```diff
@@ -14,21 +14,23 @@ const (
 	playbackID      = "abc_XYZ-123"
 )
 
+var prefixes = [...]string{"video"}
```

Comment:
I think you should add a test case for multiple prefixes `[...]string{"video", "videorec"}` in which Mist don't find anything for `video`, but it finds for `videorec`.

Reply:
since we override the `getClosestNode` method, it always returns `video` as the prefix from the loop